### ISSUE TITLE
[Feature] may support Mistral-7B recently? whatever Turbomind or Pytorch 

### ISSUE THREAD

**Issue by varuy322** [InternLM/lmdeploy#534]
### Motivation

The Mistral-7B-v0.1 Large Language Model (LLM) is a pretrained generative text model with 7 billion parameters. Mistral-7B-v0.1 outperforms Llama 2 13B on all benchmarks as offical said.
We want to calculate the output logits, when compared to llama2 and other open sourced llm.
Thanks a lot!

### Related resources

https://huggingface.co/mistralai/Mistral-7B-v0.1
